src/scripts/main.js

```diff
@@ -1,3 +1,335 @@
+        break;
+
+      case 'Salary':
+        const normalize = (item) => item.slice(1).split(',').join('');
+
+        compareResult = normalize(contentA) - normalize(contentB);
+        break;
+
+      default:
+        break;
+    }
+
+    if (sortDirection === 'desc') {
+      compareResult *= -1;
+    }
+
+    return compareResult;
+  });
+
+  sortDirection = sortDirection === 'asc' ? 'desc' : 'asc';
+
+  table.append(...data);
+});
+
+const rows = document.querySelectorAll('tr');
+let selectedRow = null;
+
+rows.forEach((row) => {
+  row.addEventListener('click', () => {
+    if (selectedRow) {
+      selectedRow.classList.remove('active');
+    }
+    row.classList.add('active');
+    selectedRow = row;
+  });
+});
+
+function removeSelect() {
+  const selected = table.querySelectorAll('.active');
+
+  for (const elem of selected) {
+    elem.classList.remove('active');
+  }
+}
+
+body.insertAdjacentHTML('beforeend', `
+  <form action="./" class="new-employee-form" name="employee-form">
+    <label>Name:
+      <input name="name" data-qa="name" type="text" required>
+    </label>
+    <label>Position:
+      <input name="position" data-qa="position" type="text" required>
+    </label>
+    <label>Office:
+      <select name="office" data-qa="office" required>
+        <option value="Tokyo">Tokyo</option>
+        <option value="Singapore">Singapore</option>
+        <option value="London">London</option>
+        <option value="New York">New York</option>
+        <option value="Edinburgh">Edinburgh</option>
+        <option value="San Francisco">San Francisco</option>
+      </select>
+    </label>
+    <label>Age:
+      <input name="age" data-qa="age" type="number" required>
+    </label>
+    <label>Salary:
+      <input name="salary" data-qa="salary" type="number" required>
+    </label>
+    <button type="submit">Save to table</button>
+  </form>
+`);
+
+const form = document.forms['employee-form'];
+const lastElem = form.elements[form.elements.length - 1];
+const firstElem = form.elements[0];
+
+lastElem.onkeydown = function(e) {
+  if (e.key === 'Tab' && !e.shiftKey) {
+    firstElem.focus();
+
+    return false;
+  }
+};
+
+firstElem.onkeydown = function(e) {
+  if (e.key === 'Tab' && e.shiftKey) {
+    lastElem.focus();
+
+    return false;
+  }
+};
+
+lastElem.addEventListener('click', e => {
+  e.preventDefault();
+
+  const data = Object.fromEntries(new FormData(form).entries());
+
+  if (!validationsOfInformation(data)) {
+    return;
+  }
+
+  saveDataToTable(data);
+  removeSelect();
+  form.reset();
+});
+
+function saveDataToTable(data) {
+  const tr = document.createElement('tr');
+
+  tr.innerHTML = `
+    <td>${data.name}</td>
+    <td>${data.position}</td>
+    <td>${data.office}</td>
+    <td>${data.age}</td>
+    <td>$${Number(data.salary).toLocaleString('en-US')}</td>
+  `;
+
+  table.append(tr);
+}
+
+const minAge = 18;
+const maxAge = 90;
+
+function validationsOfInformation(data) {
```

Comment:
function with a verb.
As an example: 
```suggestion
function checkIsValid(data) {
```